cpp/open3d/t/geometry/PointCloud.cpp

```diff
@@ -388,6 +388,23 @@ std::tuple<PointCloud, core::Tensor> PointCloud::RemoveRadiusOutliers(
     return std::make_tuple(pcd, valid);
 }
 
+PointCloud PointCloud::PaintUniformColor(const core::Tensor &color) const {
+    core::AssertTensorShape(color, {3});
+    core::Tensor clipped_color = color.To(GetDevice());
+    if (color.GetDtype() == core::Float32 ||
+        color.GetDtype() == core::Float64) {
+        clipped_color = clipped_color.Clip(0.0f, 1.0f);
+    }
```

Comment:
Should we consider the situation when color is `int` or `bool`?

Reply:
The visualizer supports Float32 data-type with range [0,1]. However, currently, we are allowing users to init. colors with any data-type. This is consistence with that (See SetPointColors).
If we want to add constraints or warnings in the future, they can be added in the `PointCloud::SetPointColors` function.